packages/app-desktop/gui/JoplinCloudConfigScreen.tsx

```diff
@@ -4,10 +4,14 @@ import { _ } from '@joplin/lib/locale';
 import { clipboard } from 'electron';
 import Button from './Button/Button';
 import { Fragment } from 'react';
+import { accountTypeToString } from '@joplin/lib/accountTypeUtils';
+const bridge = require('@electron/remote').require('./bridge').default;
```

Comment:
Consider importing from [`services/bridge.ts`](https://github.com/laurent22/joplin/blob/dev/packages/app-desktop/services/bridge.ts) instead — that file also supplies type information.
```suggestion
import bridge from '../services/bridge';
```